bftengine/src/bcstatetransfer/RangeValidationTree.hpp

```diff
@@ -365,6 +374,19 @@ class RangeValidationTree {
   static constexpr uint64_t magic_num_{0x1122334455667788};
   static constexpr uint8_t kDefaultRVBLeafLevel = 0;
   static constexpr uint8_t kDefaultRVTLeafLevel = 1;
+
+ protected:
+  concordMetrics::Component metrics_component_;
+  struct Metrics {
+    GaugeHandle rvt_size_in_bytes_;
+    concordMetrics::GaugeHandle total_rvt_nodes_;
+    concordMetrics::GaugeHandle total_rvt_levels_;
+    concordMetrics::GaugeHandle rvt_min_rvb_id_;
+    concordMetrics::GaugeHandle rvt_max_rvb_id_;
+    concordMetrics::GaugeHandle serialized_rvt_size_;
+    concordMetrics::CounterHandle rvt_validation_failures_;
```

Comment:
you have a using above, why are you using the full name then?
line 381 uses GaugeHandle and then all the rest + metrics_component_ uses concordMetrics::

Reply:
Thanks. Changed accordingly.